jack.py

```diff
@@ -674,6 +675,38 @@ def transport_stop(self):
         """Stop JACK transport."""
         _lib.jack_transport_stop(self._ptr)
 
+    def set_timebase_callback(self,
```

Comment:
Please move the method to the other callback setters.
Probably as the last one after `set_xrun_callback()`?